lib/bootloader_pvm.pm

```diff
@@ -151,20 +151,21 @@ sub boot_hmc_pvm {
     my $hmc_machine_name = get_required_var('HMC_MACHINE_NAME');
     my $lpar_id          = get_required_var('LPAR_ID');
     my $hmc              = select_console 'powerhmc-ssh';
+    my $max_wait_time    = 6;
 
     # detach possibly attached terminals - might be left over
     type_string "rmvterm -m $hmc_machine_name --id $lpar_id && echo 'DONE'\n";
     assert_screen 'pvm-vterm-closed';
 
     # power off the machine if it's still running - and don't give it a 2nd chance
-    type_string "chsysstate -r lpar -m $hmc_machine_name -o shutdown --immed --id $lpar_id && echo 'LPAR SUCCESSFULLY SHUT DOWN'\n";
+    type_string "chlparstate -m $hmc_machine_name -o shutdown --id $lpar_id -w $max_wait_time && echo 'LPAR SUCCESSFULLY SHUT DOWN'\n";
```

Comment:
this change is not needed, as we need the lpar to be shut down as soon as possible.